<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>697</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">

   <!-- INPUT DIRECTORY -->
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Input Directory</string>
     </property>
     <layout class="QFormLayout" name="formLayoutInputs">
      <item row="0" column="0">
       <widget class="QLabel" name="labelInput">
        <property name="text">
         <string>Subject</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="inputLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="browseInputButton">
        <property name="text">
         <string>Select Input Subject</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- OUTPUT DIRECTORY -->
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Output Directory</string>
     </property>
     <layout class="QFormLayout" name="formLayoutOutputs">
      <item row="0" column="0">
       <widget class="QLabel" name="labelOutput">
        <property name="text">
         <string>Directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="outputLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="browseOutputButton">
        <property name="text">
         <string>Select Output</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- RABBIT SECTION -->
   <item>
    <widget class="ctkCollapsibleButton" name="rabbitCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Rabbit</string>
     </property>
     <layout class="QVBoxLayout" name="layoutRabbit">

      <!-- IN VIVO -->
      <item>
       <widget class="QGroupBox" name="groupInvivo">
        <property name="title">
         <string>In vivo</string>
        </property>
        <layout class="QFormLayout" name="formLayoutInvivo">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxRabbitInvivoParenchyma">
           <property name="text">
            <string>Parenchyma</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBoxRabbitInvivoParenchymaAirways">
           <property name="text">
            <string>Parenchyma/Airways</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxRabbitInvivoAirways">
           <property name="text">
            <string>Airways</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBoxRabbitInvivoParenchymaAirwaysVascularTree">
           <property name="text">
            <string>Parenchyma/Airways/Vascular Tree</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxRabbitInvivoVascularTree">
           <property name="text">
            <string>Vascular Tree</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxRabbitInvivoLobes">
           <property name="text">
            <string>Lobes</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>

      <!-- EX VIVO -->
      <item>
       <widget class="QGroupBox" name="groupExvivo">
        <property name="title">
         <string>Ex vivo</string>
        </property>
        <layout class="QFormLayout" name="formLayoutExvivo">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxRabbitExvivoParenchymaAirways">
           <property name="text">
            <string>Parenchyma/Airways</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxRabbitExvivoAirways">
           <property name="text">
            <string>Airways</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>

      <!-- AXIAL -->
      <item>
       <widget class="QGroupBox" name="groupAxial">
        <property name="title">
         <string>Axial</string>
        </property>
        <layout class="QFormLayout" name="formLayoutAxial">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxRabbitAxialAll">
           <property name="text">
            <string>Parenchyma/Airways/Vascular Tree</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- PIG SECTION -->
   <item>
    <widget class="ctkCollapsibleButton" name="pigCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Pig</string>
     </property>
     <layout class="QVBoxLayout" name="layoutPig">

      <!-- AXIAL -->
      <item>
       <widget class="QGroupBox" name="groupAxial">
        <property name="title">
         <string>Axial</string>
        </property>
        <layout class="QFormLayout" name="formLayoutAxial">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxPigAxialParenchyma">
           <property name="text">
            <string>Parenchyma</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- RAT SECTION -->
   <item>
    <widget class="ctkCollapsibleButton" name="ratCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Rat</string>
     </property>
     <layout class="QVBoxLayout" name="layoutRat">

      <!-- INVIVO -->
      <item>
       <widget class="QGroupBox" name="groupInvivo">
        <property name="title">
         <string>Invivo</string>
        </property>
        <layout class="QFormLayout" name="formLayoutInvivo">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxRatInvivoParenchymaAirwaysKidneysHeart">
           <property name="text">
            <string>Parenchyma/Airways/Kidneys/Heart</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- PROGRESS BAR -->
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>

   <!-- SEGMENT BUTTON -->
   <item>
    <widget class="QPushButton" name="pushButtonSegmentation">
     <property name="text">
      <string>Segment</string>
     </property>
    </widget>
   </item>

   <!-- SPACER -->
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>

 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>

 <resources/>
 <connections/>
</ui>
